### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by shaon07**
update the class syntax. previously it was give wrong class syntax. i fix it with write one.

### CODE REVIEW

**Review by Josh-Cena — CHANGES_REQUESTED** [review]
Please revert the change that added your name (or other names). That's totally unnecessary.

You need to change the code language to `js` and change "pseudocode" in the text to "JavaScript" since it's not "pseudocode" anymore.

File: files/en-us/learn/javascript/objects/object-oriented_programming/index.md [hunk 1]
```diff
@@ -67,30 +74,35 @@ On its own, a class doesn't do anything: it's a kind of template for creating co
...
 
 This constructor takes two parameters, so we can initialize the `name` and `teaches` properties when we create a new concrete professor.
 
 Now that we have a constructor, we can create some professors. Programming languages often use the keyword `new` to signal that a constructor is being called.
 
 ```js
-walsh = new Professor("Walsh", "Psychology");
-lillian = new Professor("Lillian", "Poetry");
+const shaon = new Professor("Shaon", "Psychology");
+const shamirul = new Professor("MD Shamirul Islam", "mathematician");
```
Comment (estelle) [inline files/en-us/learn/javascript/objects/object-oriented_programming/index.md:99]: Please don't include your name or the names of actual people in the documentation.

**Comment by wbamberg** [comment]
As it says at the start of the page:

> For now, we'll describe these concepts without reference to JavaScript in particular, so all the examples are given in [pseudocode](https://developer.mozilla.org/en-US/docs/Glossary/Pseudocode).

The idea is that this page is not about JS per se, but OO concepts in general. So using pseudocode is intentional.

With this, people have to understand all the JS class syntax, which we haven't taught yet.

And this PR makes the page incoherent: for example the first code sample doesn't include a constructor, because we haven't talked about them yet. But this PR adds a constructor to the first sample, which the accompanying prose doesn't then mention, and then the next bit starts randomly talking about constructors.

**Review by skyclouds2001 — COMMENTED** [review]
I think the change it not correct, as it is only pseudocode to represent the concept of OOP

**Comment by Josh-Cena** [comment]
Closing since this page is not about JavaScript.

**Pull request closed without merging**